perf/scripts/get_cpuinfo.py

```diff
@@ -1,4 +1,25 @@
 import subprocess
+import platform
+from shutil import which
+def getestimatedcpufrequency():
+    # We should be checking the architecture we are running on, but platform.processor()
+    # is "unknown" in our container
+    if platform.system() != "Linux":
+        return None
+    p = subprocess.run(["gcc", "-O2", "-o", "/opt/test/cpu_frequency_estimate", "/opt/test/cpu_frequency_estimate.c"])
```

Comment:
The fixed path in here makes (file/folder) changes in the docker file extremely difficult -- is there any way to do things differently? e.g., build the executable earlier and then copy it over to the test image with all other executables, possibly after first testing that it runs OK?

Reply:
In theory building on the host then copying it over to the image would be fine in most cases. The case that I think this could be an issue is if you build profiling on macOS. You would get a Mach-O binary rather than an ELF binary, and it would cause other issues.

I might be able to get away with using a relative path. I did an absolute path as part of debugging an issue. I'll do a quick test and I'll let you know.